changelogs/unreleased/gh-5843-fiber-wakeup.md

```diff
@@ -0,0 +1,6 @@
+## bugfix/core
+
+Remove assertion raised in case of `fiber_wkeup()` get called with
```

Comment:
Please, add `*` in the beginning like all changelogs do. This is later displayed as a list in github markdown.

Reply:
sure, updated